AppKit/CPViewController.j

```diff
@@ -176,6 +176,58 @@ var CPViewControllerCachedCibs;
         _view = [CPView new];
 }
 
+- (void)loadViewAsynchronously:(Function/*(view, error)*/)aHandler
```

Comment:
Yes, I think this is how we should name it too. I will now start using it in my own code.